docs/blocks-dynamic.md

```diff
@@ -0,0 +1,77 @@
+It is also possible to create dynamic blocks. These are blocks that can change their content even if the post is not saved. One example from WordPress itself is the latest posts block. This block will update everywhere it is used whena new post is published.
+
+The following code example shows how to create the latest post block dynamic block.
+
+```js
+// myblock.js
+
+var el = wp.element.createElement,
+	registerBlockType = wp.blocks.registerBlockType,
+	withAPIData = wp.components.withAPIData;
+
+registerBlockType( 'riad/latest-post', {
```

Comment:
Per consistency with other docs, should we use a more generic namespace here, since without context of blog it's sourced from, unclear what Riad's name is doing here 😄  Other places we're using `my-plugin`.